docs/source/api/v2/ping.rst

```diff
@@ -0,0 +1,62 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-ping:
+
+********
+``ping``
+********
+Checks whether Traffic Ops is online.
+
+``GET``
+=======
+:Auth. Required: No
+:Response Type:  Object
+
+Request Structure
+-----------------
+No parameters available.
+
+.. code-block:: http
+	:caption: Request Example
+
+	GET /api/2.0/ping HTTP/1.1
+	User-Agent: python-requests/2.22.0
+	Accept-Encoding: gzip, deflate
+	Accept: */*
+	Connection: keep-alive
+	Cookie: mojolicious=...
+
+Response Structure
+------------------
+
```

Comment:
This should just say something like "returns an object containing only the `"ping"` property which always has the value `"pong"`.

Reply:
Updated definition in 8fe4ab7d06.